Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -115,6 +115,30 @@ - (NSURL *)getUrlForGetAction:(NSString *)action withArgs:(NSDictionary *)args
 /**
  @method
  
+ @abstract Percent encodes the url query parameters
+ 
+ @param paramValue The value to encode
+ 
+ @return an `NSString` containing the full URL
+ */
+- (NSString *)encodeURLParam:(NSString *)paramValue
+{
+    if ([paramValue isKindOfClass:[NSString class]])
+    {
+        if (subSet == nil) {
+            NSMutableCharacterSet* workingSet = [[NSCharacterSet URLQueryAllowedCharacterSet] mutableCopy];
+            [workingSet removeCharactersInString:@"+"];
+            subSet = [workingSet copy];
+        }
+        paramValue = [paramValue stringByAddingPercentEncodingWithAllowedCharacters:subSet];
```

Comment:
i would prefer to see `return [paramValue stringByAddingPercentEncodingWithAllowedCharacters:subSet];` here, and an `else` case that returns `paramValue`